src/chief.t.sol

```diff
@@ -176,42 +181,35 @@ contract DSChiefTest is DSThing, DSTest {
         chief.launch();
     }
 
-    function testFail_enable_system() public {
+    function test_enable_system() public {
```

Comment:
IMO this could have a more descriptive name, e.g. `test_launch_threshold` or similar.